data/json/scenarios.json

```
@@ -377,8 +377,8 @@
     "id": "alcatraz",
     "name": "Challenge - Island Prison",
     "points": -6,
-    "description": "You were at a high-security prison right before the Cataclysm.  You managed to make it outside the inner walls…  Too bad it's located on a remote island, and now you need to find out how to escape it too.",
-    "allowed_locs": [ "sloc_prison_alcatraz" ],
+    "description": "You were in a high-security prison right before the Cataclysm.  Unfortunately, you're still in it, so you need to figure out how to escape.  It's located on a remote island to boot; you'll eventually need to escape that also.",
```

```suggestion
    "description": "You were in a high-security prison right before the Cataclysm.  Unfortunately, you're still in it, so you need to figure out how to escape.  (It's located on a remote island to boot; you'll eventually need to escape that also.)",
```
Is this (starting inside) true of the `sloc_prison_alcatraz` also?

---

Nope, on `alcatraz` variant you're starting outside.

---

Urr. In that case, there really need to be two different descriptions (probably meaning two different Island Prison challenges).

---

Unfortunately, this is not possible. There could be only one description for the scenario. This is why we need something neutral.
Also it would be very helpful if we could set different scenario descriptions based on scenario professions.

---

```suggestion
    "description": "You were in a high-security prison right before the Cataclysm.  You're in a better position to escape than you were before, but you're not there yet.  For one thing, it's located on a remote island to boot.",
```

---

```suggestion
    "description": "You were in a high-security prison right before the Cataclysm.  This is a problem, particularly since it's located on a remote island.",
```

---

I liked your first variant, and without parentheses.

---

I like how it reads, but I'm not sure how to adapt it for the Alcatraz setup, or non-prisoner profession.

---

It's already neutral enough to fit both alcatraz and this setup, and all available professions.